libs/application/templates/estate/src/lib/utils/customZodError.ts

```diff
@@ -0,0 +1,15 @@
+import { MessageDescriptor } from 'react-intl'
+import { ZodTypeAny } from 'zod'
+
+export const customZodError = (
```

Comment:
Is this a utility function to make the error messages easier to write or is it adding any additional functionality to the zod validation, maybe both? Thinking this might cause hard to find errors if validation changes as these custom functionality might be missed on validation improvements.
Should it be added as a validation util in the core instead?

Reply:
You're right on that. I'm putting it on my tasklist and will address this in a special pull requests since it affects two applications currently. Thank you :)